### PR TITLE
User auth

### PR DESCRIPTION
**Pull request by shushengfuqin**
### Summary 
using google to login.

### Details
In the login page, there is a google login button. Which will login with the googleId. it's already unique. 

### How to test?
Run and login with google account then try create. Log out, login again with google and see if it's still there. 

### Checks 
- [ ] Tested Changes
- [ ] Stakeholder Approval

### CODE REVIEW

File: src/util/customHooks/googleLoginFunc.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    alert(`Failed to login.`);
+  };
+
+  return {
+    onSuccess,
+    onFailure,
+  };
+};
+
+function removeSpace(x: string){
```
[the remark this answers is not in the document]
Comment (shushengfuqin) [reply src/util/customHooks/googleLoginFunc.tsx]: solved

**Review by mckenzietsean — APPROVED** [review]
Everything looks good to me.

**Review by JustinLohKingWei — COMMENTED** [review]
Untitled
Login works fine although you might want to check if localhost :3000 may change when added to deployment server. Giving my approval due to no other concerns

**Review by Ysabz — COMMENTED** [review]
works well on localhost but on static web it gives authorization error

**Pull request merged**